loadtest/report/compare.go

```diff
@@ -104,9 +106,12 @@ func calculateDeltas(reports ...Report) comp {
 		// XXX: This can be somewhat refactored but whether absolute metrics
 		// are useful or not needs to be seen.
 		for label, value := range base.AvgStoreTimes {
-			actual := roundTo3DecimalPlaces(float64(r.AvgStoreTimes[label]))
-			delta := actual - roundTo3DecimalPlaces(float64(value))
-			deltaP := (delta / actual) * 100
+			actual := getDuration(float64(r.AvgStoreTimes[label]))
+			delta := actual - getDuration(float64(value))
+			deltaP := (delta.Seconds() / actual.Seconds()) * 100
+			if math.IsNaN(deltaP) {
+				deltaP = 0
+			}
```

Comment:
The other option would be not to bother calculating `deltaP` if `delta` is zero.

Reply:
Yeah .. that was my first approach. Somehow didn't like the code arrangement. :P Plus I get to use an esoteric stdlib function. :)